docs/backend/guides/how-to/add-an-experimental-feature-flag.md

```diff
@@ -0,0 +1,118 @@
+---
+sidebar_position: 7
+---
+
+# How to set up an experimental feature flag
+
+Backend feature flags via the "experimental" way utilize environment variables rather than Flipt API or Chamber secrets.
+
+## Setting up environment variables
+
+### Naming convention
+
+Feature flags stored inside of the environment variables files should be prefixed with `FEATURE_FLAG_`. It must be prefixed this way in order for the `EnvFetcher` to properly retrieve it.
+
+Code example:
+`export FEATURE_FLAG_EXAMPLE=true` (Set's the feature flag *`example`* to true)
+
+### CircleCI deployment flags
+
+To set a feature flag as enabled or disabled within certain deployment environments, you must update the corresponding [config](https://github.com/transcom/mymove/tree/main/config) environment variables. CircleCI deployment environments can be found under the [config/env folder](https://github.com/transcom/mymove/tree/main/config/env).
+
+When you export your feature flag variable for a specific environment, make sure you update its "app" and its respective "app-client-tls" file. Example:
+
+```
+// ${environment} can be "prd", "exp", "stg", and so on.
```

Comment:
Can you also list `loadtest` and `demo` environments explicitly instead of "and so on"?

Reply:
You bet!

Reply:
Also put in the "review" environment which apparently has a `.env` file